extras/data/Makefile

```diff
@@ -2,7 +2,7 @@ PY=python3
 
 PACKAGE=machinery
 OUTPUT=_runcards
-IGNORE=Makefile ${PACKAGE} README.md ${OUTPUT} banana.yaml benchmark.db
+IGNORE=Makefile ${PACKAGE} README.md ${OUTPUT} banana.yaml benchmark.db navigator.py __pycache__
```

Comment:
Actually... why do we need a `navigator.py`?

Reply:
you put it back in the days when you first wrote the data part ... maybe nowadays we can configure the navigator by CLI arguments? because here we're running our own database

Reply:
Uhm, I guess you're right. The only non-trivial thing it is doing the script is:
```py
banana.register(".")
```
and now we can replace by `yadnav --config` argument.

So, maybe we can kill the script and remove this mention in the `Makefile` as well.